src/library/Connect/WorkloadInformationProvider.cs

```diff
@@ -509,7 +509,7 @@ private int GetServicePortValue(V1ServicePort servicePort, V1Container container
                             DnsName = dns,
                             Ports = subset.Ports?
                                 .Where(port => this._IsSupportedProtocol(port.Protocol, endpoint.Metadata.Name) && !(portToIgnoreForHeadlessServiceEndpoints.GetValueOrDefault(endpoint.Metadata.Name)?.Contains(port.Port) ?? false))
-                                .Select(p => new PortPair(remotePort: p.Port, p.Name))
+                                .Select(p => new PortPair(remotePort: p.Port,name : p.Name))

```

Comment:
this was causing named port as empty or null which didn't set the env variable [here](https://github.com/Azure/Bridge-To-Kubernetes/blob/main/src/library/Connect/LocalEnvironmentManager.cs#LL501C21-L501C21)